server/src/frontend_api.rs

```diff
@@ -768,10 +785,12 @@ mod tests {
                 "*:development.03fa5f506428fe80ed5640c351c7232e38940814d2923b08f5c05fa7",
             ))
             .to_request();
+        let r = test::call_service(&app, req).await;
+        debug!("{}", r.status());
+        /*        let result: FrontendResult = test::call_and_read_body_json(&app, req).await;
```

Comment:
If we don't need it anymore, can we yeet it? Or is this something that's helpful to uncomment for testing the tests?